content/features/proxy2/configure.md

```diff
@@ -0,0 +1,411 @@
+
+* `debug(1)` turns on a debugging mode that is mainly useful if you are modifying the routing library. This setting also lets you use the `dsay({{<var>}}TEXT{{</var>}})` function to print <var>TEXT</var> to standard output while loading the proxy configuration.
+
+### Adjust proxy settings
+
+To fine-tune various proxy behaviors, include a `settings{}` block in your configuration file. The block contains any number of function calls that set values on the `mcp` object. For the full list of available setting functions, see [General API documentation]({{<legacy_proxy_base_path>}}#general-api-documentation).
+
+For example, the following settings block sets a ten-second connection timeout, and an upper limit of five active requests:
+
+```lua
+settings {
+    mcp.backend_connect_timeout(10)
+    mcp.active_req_limit(5)
+}
+```
+
+### Define backend pools
+
+Use the required `pools{}` block to define the locations and other attributes of the back-end Memcached servers that the router directs queries to.
+
+The `pools{}` block contains one or more pool definitions in the following format:
+
+```lua
+{{<var>}}POOL_NAME{{</var>}} = {
+    backend_options = { {{<var>}}BACKEND_OPTIONS{{</var>}} },
+    backends = {
+        "{{<var>}}HOST{{</var>}}:{{<var>}}PORT{{</var>}}",
+        {{<var>}}[ ... ]{{</var>}}
+    }
+}
+```
+
+Replace the following:
+
+* <var>POOL_NAME</var>: a name for this pool—for example, `northeast_1`.
+* <var>BACKEND_OPTIONS</var>: (optional) a comma-separated list, in `key=value` format, of settings for the proxy to apply to all of the backends in the pool. The proxy applies for the default values for any settings not defined here. For a list of settings, see [Backend API]({{<legacy_proxy_base_path>}}/Proxy#backend-api).
+* <var>HOST</var>: the IP address or hostname of the Memcached server you are defining as a backend.
+* <var>PORT</var>: (optional) the TCP port of the Memcached server you are defining as a backend. The default value is `11211`.
+
+To fine-tune the proxy settings for an individual backend, replace its entry in the `backends` list with a curly-braced list of back-end options. The list must include `host`.
+
+The following example defines two backends, including one that runs on port 11212 and has a ten-second connection timeout:
+
+```lua
+backends = {
+    "127.0.0.1:11211",   
+    { host = "127.0.0.1", port = 11212, connecttimeout = 10 },
+}
+```
+
+You can optionally gather pools into a named _set_, which contains a list of pool definitions. You might want to employ this technique if you have several pools of backends that you want the router to treat as mutually equivalent with synchronized data:
+
+```lua
+{{<var>}}SET_NAME{{</var>}} = {
```

Comment:
Do the examples need to be capitalized? I think routelib actually requires lowercase `set_` and people might give everything uppercase names. maybe I could fix that in the library tho...

Reply:
I've added clarity around set definition with https://github.com/memcached/memcached.github.io/pull/3/commits/7e99460cf4d6f43ffcc669ffe20a8e2a76fe098a, including moving it into its own sub-section and making the naming rules more clear.